turnpike/plugins/x509.py

```diff
@@ -0,0 +1,28 @@
+import logging
+from flask import request
+
+from ..config import HEADER_CERTAUTH_SUBJECT, HEADER_CERTAUTH_ISSUER
```

Comment:
Doing the import of the config this way would prevent installers from overriding with their own config file. What you probably want to do is something like:

```
from flask import current_app

class X509AuthPlugin(TurnpikeAuthPlugin):
    # ... snip ...
    @property
    def headers_needed(self):
        return set([current_app.config.HEADER_CERTAUTH_SUBJECT, current_ap.config.HEADER_CERTAUTH_ISSUER])
```

Does that make sense?